articles/realtime.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: default
+title: ROS 2.0 in Realtime
+permalink: articles/realtime.html
+abstract: This article describes real-time computing requirements and how ROS 2 will fulfill them.
+published: false
+author: Jackie Kay
+---
+
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
```

Comment:
It would be good to start a summary of the document, stating its scope, goal and structure.